src/course_bot/quiz.clj

```diff
@@ -1,10 +1,11 @@
 (ns course-bot.quiz
   (:require [clojure.string :as str])
   (:require [codax.core :as codax])
-  (:require [course-bot.general :as general :refer [tr]]
+  (:require [course-bot.internationalization :as i18n :refer [tr, normalize-yes-no-text]]
```

Comment:
Better to use `i18n/normalize-yes-no-text`. Here and in other places.

Reply:
Okey, no problems. But can you explain why? Just interesting

Reply:
The programmer see i18n and immediately understand that it is about localisation. Also, it don't make a mess from the code (what will happen if we add it to `tr` function).